.github/workflows/samples-python-petstore.yaml

```diff
@@ -0,0 +1,59 @@
+name: "Python Client: Petstore"
+
+on:
+  pull_request:
+    paths:
+      - samples/openapi3/client/petstore/python-aiohttp/**
+      - samples/openapi3/client/petstore/python/**
+      - .github/workflows/samples-python-petstore.yaml
+
+jobs:
+  build:
+    name: Test Python client
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          #- "3.11"
+        sample:
+          - samples/openapi3/client/petstore/python-aiohttp
+          - samples/openapi3/client/petstore/python
+    services:
+      petstore-api:
+        image: swaggerapi/petstore
+        ports:
+          - 80:8080
+        env:
+          SWAGGER_HOST: http://petstore.swagger.io
+          SWAGGER_BASE_PATH: /v2
```

Comment:
I think you will also need to modify the host table to point petstore.swagger.io to 127.0.0.1 in another github action `steps`

Reply:
saw you updated HOST below.